python/pyspark/sql/types.py

```diff
@@ -899,6 +899,8 @@ def _parse_datatype_json_value(json_value):
             return _all_atomic_types[json_value]()
         elif json_value == 'decimal':
             return DecimalType()
+        elif json_value == 'timestamp_ntz':
+            return TimestampType()
```

Comment:
I think we need a design for how to support TimestampNTZ type in PySpark. Here the `timestamp_ntz` is mapped to `TimestampType()`
cc @HyukjinKwon

Reply:
Yeah, let's add the proper support of `TimestampTypeNTZ` first.

Reply:
Updated with new TimestampNTZType